README.md

```diff
@@ -13,3 +13,54 @@ The repo contains 1152 message declarations and 32 enum declarations in 192 `.pr
 This includes command, event, rejection, entity, ID, and value object types.
 The declarations are split into several identical packages which only differ in names.
 This allows us to have more data for testing build tool performance.
+
+## Manual usage
+
+To use this repo locally for one-time tests, follow these steps.
+
+1. Add the repo as a Git submodule to your project.
+2. Copy the `settings.gradle.kts.template` file as `settings.gradle.kts`.
+   Replace all the placeholder versions with the required versions. Or, alternatively,
+   add a `dependencySubstitution` block and define which local projects should be used for which
+   dependencies.
+3. Build the project with the `build` task.
+
+## Using with Gradle
+
+To launch the build performance tests with Gradle, follow these steps.
+
+1. Add the repo as a Git submodule to your project.
+2. Create a Gradle task which would call the `substitute-settings.py` script, setting the versions
+   to the versions from the local environment:
+```kotlin
+val prepareBuildPerformanceSettings by tasks.registering(Exec::class) {
+    environment(
+        "MC_JAVA_VERSION" to Spine.McJava.version,
+        "CORE_VERSION" to Spine.ArtifactVersion.core,
+        "PROTO_DATA_VERSION" to ProtoData.version,
+        "VALIDATION_VERSION" to Validation.version
+    )
+    workingDir = File(rootDir, "BuildPerformance")
```

Comment:
We have `buildSrc` in all the projects. We'd probably migrate it to be `build-logic`(via `includeBuild`)  as some of the people do to improve the build performance. But even if we do, it' probably going to be `build` from a lowercase `b`.

Why do we want `BuildPerformance` coming from capital `B`?